10MinuteMail.net/Types/MailContent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace _10MinuteMail.net.Types
+{
+    [Serializable]
+    public class MailContent
+    {
+        [JsonProperty(PropertyName = "from")]
+        public string From { get; set; }
+        [JsonProperty(PropertyName = "to")]
+        public string To { get; set; }
+        [JsonProperty(PropertyName = "gravatar")]
+        public string Avatar { get; set; }
+        [JsonProperty(PropertyName = "subject")]
+        public string Subject { get; set; }
+        [JsonProperty(PropertyName = "datetime")]
+        public string Datetime { get; set; }
```

Comment:
Naming is a little inconsistent. Some properties are named `DateTime` and some `Datetime`. It would be nice if you could rename all of them to `DateTime`.

```suggestion
        public string DateTime { get; set; }
```